### PR TITLE
Readme lints

### PR DESCRIPTION
**Pull request by pablodeymo**
Markdown lints in the README

### CODE REVIEW

**Review by gabrielbosio — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -654,198 +659,199 @@ Now we will create our `Initialize` method step by step:
...
 
-*Compute operands*
+##### Compute operands
 
 ```go
     //TODO
 ```
 
-*Opcode assertions*
+##### Opcode assertions
```
Comment (fmoletta) [inline README.md]: We should move these sections to the `Instruction Decoding and Execution` section (line 493)

File: README.md [hunk 2]
```diff
@@ -945,118 +951,117 @@ And we are all set to integrate this new logic into our `Memory`'s `Insert` oper
...
+    r.executionBase = r.Vm.Segments.AddSegment()
+    // Builtin Segments
+    for i := range r.Vm.BuiltinRunners {
+        r.Vm.BuiltinRunners[i].InitializeSegments(&r.Vm.Segments)
+    }
 }
 ```
 
-*InitializeMainEntryPoint*
+##### InitializeMainEntryPoint
```
Comment (fmoletta) [inline README.md]:
```suggestion
##### InitializeMainEntryPoint(Builtins)
```
Keep consistency with other sub-headers in the same section

File: README.md [hunk 3]
```diff
@@ -945,118 +951,117 @@ And we are all set to integrate this new logic into our `Memory`'s `Insert` oper
...
+    m.data[addr] = *val
+
+    return m.validateAddress(addr)
 }
 ```
 
 Now we will initialize the builtins from our `CairoRunner`:
 
-*NewCairoRunner*
+##### NewCairoRunner
```
Comment (fmoletta) [inline README.md]:
```suggestion
##### NewCairoRunner (Builtins)
```
Keep consistency with other sub-headers in the same section

File: README.md (hunk 1 above)
Comment (pablodeymo) [reply README.md]: done

Comment (fmoletta) [reply README.md]: My bad! It was line 552 (line 493 is before the Write your own vm section, in the general overview section)

Comment (pablodeymo) [reply README.md]: done!

**Review by fmoletta — APPROVED** [review]

**Pull request merged**